core/client/app/mixins/pagination.js

```diff
@@ -51,11 +65,15 @@ export default Mixin.create({
 
         paginationSettings.page = 1;
 
+        this.set('isLoading', true);
+
         return this.get('store').query(modelName, paginationSettings).then((results) => {
             this.set('paginationMeta', results.meta);
             return results;
         }, (response) => {
```

Comment:
Not that it's a huge deal, but would it be better to use `catch` here, just to stay consistent?